src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Main from './Components/Main';
 import './App.css';
```

Comment:
As we're not using CSS files for styling it's better to remove the import of `./App.css` (also in `index.tsx`) and also the file itself.